docs/SPEC.md

```diff
@@ -102,7 +102,7 @@ The following character sequences represent operators:
     -   <    !~   [   ]   ^
     *   >    =~   {   }   ?
     /   <=   =    ,   :   "
-    %   >=   <-   .   |>
+    %   >=   <-   .   |>  @
```

Comment:
I think we should avoid calling this an operator but also why are quotes in here too?

We might want to review exactly what we're trying to communicate with this section. It seems we're trying to say that these are characters that have special meaning lexically but calling them operators.

Reply:
yeah its a bit confusing, these are the set of lexical tokens. Perhaps we should refactor that section.